### PR TITLE
ddl: Remove `TODO` about check all versions to handle the waiting time problem

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: ddl/ddl_worker.go [hunk 1]
```diff
@@ -348,6 +340,30 @@ func (d *ddl) waitSchemaChanged(waitTime time.Duration, latestSchemaVersion int6
 	return
 }
 
+// handleException handles the following situation:
+// If the job enters a new state, and the worker crashs when it's in the process of waiting for 2 * lease time,
+// Then the worker restarts quickly, we may run the job immediately again,
+// but in this case we don't wait enough 2 * lease time to let other servers update the schema.
+// So here we get the latest schema version to make sure all servers' schema version update to the latest schema version
+// in a cluster, or to wait for 2 * lease time.
+func (d *ddl) handleException(job *model.Job, waitTime time.Duration) {
```
Comment (winkyao) [inline ddl/ddl_worker.go]: name it `waitSchemaSynced`?

**Comment by zimulala** [comment]
PTAL @winkyao

**Comment by zimulala** [comment]
PTAL @shenli

File: ddl/syncer.go [hunk 2]
```diff
@@ -182,6 +184,30 @@ func (s *schemaVersionSyncer) RemoveSelfVersionPath() error {
...
+			continue
+		}
+		if err == nil && len(resp.Kvs) > 0 {
+			var ver int
+			ver, err = strconv.Atoi(string(resp.Kvs[0].Value))
+			if err == nil {
+				return int64(ver), nil
+			}
+		}
+		log.Infof("[syncer] get global version failed %v", err)
```
Comment (shenli) [inline ddl/syncer.go]: There will be a lot of logs.

**Comment by zimulala** [comment]
PTAL @shenli

**Comment by zimulala** [comment]
PTAL @shenli

**Comment by shenli** [comment]
LGTM

**Pull request merged**